modular_zubbers/code/modules/clothing/head/wig.dm

```diff
@@ -0,0 +1,73 @@
+/obj/item/clothing/head/wig
+	var/obj/item/clothing/head/head_accessory
+	var/mutable_appearance/head_accessory_MA
+
+	var/overlay_on = FALSE
+
+	var/mob/living/carbon/holder
+	var/item_path
+
+/obj/item/clothing/head/wig/attackby(obj/item/I, mob/living/user)
+	. = ..()
+	//Checks if the item is in the list of items available
+	for(var/type in HEAD_ACCESSORIES_PATHS)	
+		if(istype(I, type))
+			item_path = type
+			break
+
+	if(item_path && !head_accessory)
+		add_head_accessory(I)
+	else
+		to_chat(user, span_notice("You can't put \the [I.name] on the head of \the [holder.name]"))
+		return
+
+/obj/item/clothing/head/wig/proc/add_head_accessory(obj/item/clothing/I)
+	//Get the mutable_appearance ready
+	head_accessory_MA = mutable_appearance(I.worn_icon)
+	head_accessory_MA.icon_state = I.icon_state
+	I.forceMove(src) //Put the item in the wig
+
+	if(holder)
+		holder.add_overlay(head_accessory_MA)
+		overlay_on = TRUE
+
+	head_accessory = I
+
+/obj/item/clothing/head/wig/attack_hand_secondary(mob/user)
+	..()
+	. = SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
```

Comment:
Are you sure this is correct & what lead you to this?

Reply:
Uh well, here I go through the parent conditions and then just cancel the attack chain...
Is there anything wrong with that snippet?

Reply:
Sure, sounds good. 
Might need a
```
	if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)
		return```
Which is a pattern that shows up occassionally when searching, I can't tell, would have to ask coderbus. But this is fine to merge as is.